### DISCUSSION TITLE
Possible bug: Various utilities not being compiled with multiple config files

### DISCUSSION THREAD

**Discussion by robdekort** [tailwindlabs/tailwindcss#6310] [Help]
Hey folks,

In v3 I'm having issues getting the svg utilities and (plugin based) aspect utilities compiled. 

You can easily reproduce this with this simple bare project: [TW3.zip](https://github.com/tailwindlabs/tailwindcss/files/7691184/TW3.zip)

Run `npm i && npm run watch`

Note that the `aspect-w-*`, `aspect-h-*`, `stroke-current` and `fill-current` utilities are not being compiled into `site.css`. However when you comment out line 5 in `tailwind.config.js` it starts working.

This wasn't an issue for me in v2.  

Could this be a bug or was it never intended to work like this?

Thanks!

**Comment by browner12** [comment]
The aspect ratios have been pulled into the main project, so you don't need the plugin anymore.

**Reply by robdekort** [reply]
Well for backwards compatibility you can still pull it in. That way you get the old padding trick utilities so you can support Safari 14.

**Reply by sjclark** [reply]
> The aspect ratios have been pulled into the main project, so you don't need the plugin anymore.

I'm all for coding things as future-facing as possible, but when 1 in 5 visitors will get code that doesn't work it's probably a bit much (https://caniuse.com/mdn-css_properties_aspect-ratio)

**Comment by robdekort** [comment]
It seems somehow the config files overrule each other.

**Comment by robdekort** [comment]
Ok I solved most of my issues. I didn't read properly as you have to explicitly define `currentColor` now when overwriting the color config.

**Comment by robdekort** [comment]
However the aspect utilities still don't get compiled when you have multiple config files:

* tailwind.config.js (requiring the following two files)
* tailwind.config.1.js
* tailwind.config.2.js 

They only get compiled when you require the Aspect Ratio plugin in `tailwind.config.2.js` (see zip from initial post).
